emacs/agda/LICENSE

```diff
@@ -0,0 +1,59 @@
+Copyright (c) 2005-2015 Ulf Norell, Andreas Abel, Nils Anders
+Danielsson, Andrés Sicard-Ramírez, Dominique Devriese, Péter
+Divianszki, Francesco Mazzoli, Stevan Andjelkovic, Daniel Gustafsson,
+Alan Jeffrey, Makoto Takeyama, Andrea Vezzosi, Nicolas Pouillard,
+James Chapman, Jean-Philippe Bernardy, Fredrik Lindblad, Nobuo
+Yamashita, Fredrik Nordvall Forsberg, Patrik Jansson, Guilhem Moulin,
+Stefan Monnier, Marcin Benke, Olle Fredriksson, Darin Morrison, Jesper
+Cockx, Wolfram Kahl, Catarina Coquand
+
+Permission is hereby granted, free of charge, to any person obtaining
+a copy of this software and associated documentation files (the
+"Software"), to deal in the Software without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so, subject to
+the following conditions:
+
+The above copyright notice and this permission notice shall be
+included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+--------------------------------------------------------------------------------
+
+The file src/full/Agda/Utils/ReadP.hs is Copyright (c) The University of
```

Comment:
The second half of this file can be removed, since it concerns a file we do not ship.